src/s3/commands/downloadFileAs.ts

```diff
@@ -20,7 +21,12 @@ import { streamToBuffer, streamToFile } from '../../shared/utilities/streamUtili
 import { S3File } from '../fileViewerManager'
 import globals from '../../shared/extensionGlobals'
 import { telemetry } from '../../shared/telemetry/telemetry'
+import { S3FolderNode } from '../explorer/s3FolderNode'
+import { isNonNullable } from '../../shared/utilities/tsUtils'
+import { getLogger } from '../../shared/logger'
+import * as localizedText from '../../shared/localizedText'
 
+const downloadBatchSizeLimit = 20
```

Comment:
nit: this isn't really a limit?
```suggestion
const downloadBatchSize = 20
```